contrib/scraper_sao_luis_contact.py

```
@@ -0,0 +1,104 @@
+
+        NOTE: This website uses JavaScript to load parish data dynamically.
+        If this scraper doesn't work, it will need to be updated to use
+        scrapy-playwright or scrapy-selenium to render JavaScript.
+        """
+        parish_id = response.url.split("paroquiaid=")[-1]
+
+        parish_name = response.css(".paroquia-details .name::text").get()
+        if not parish_name:
+            parish_name = response.css("title::text").get()
+            if parish_name:
+                parish_name = parish_name.replace("–", "-").split("-")[0].strip()
```

Ruff (0.14.5)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)